### PR TITLE
Added references to learn material

### PR DESCRIPTION
**Pull request by TrudAX**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -67,6 +67,11 @@ Inspired by [awesome-python](https://github.com/vinta/awesome-python).
 
 *Resources with good learn material*
 
+* [Microsoft learn](https://docs.microsoft.com/en-us/learn/browse/?expanded=dynamics-365&products=dynamics-finance-operations%2Cdynamics-scm&roles=developer) - Official free Microsoft Dynamics 365FO trainings 
+* [Dynamics 365 Success by Design Implementation Guide](https://www.d365implementationguide.com/books/asvr/#p=1) - Dynamics 365 Implementation Guide interactive book
+* [ExamTopics - Expert Verified, Online, Free certification questions](https://www.examtopics.com/exams/microsoft/mb-300/) - a lot of free certification questions to prepare to D365FO related certifications
```
Comment (anderson-joyle) [inline README.md]: This website is really cool but this particular link leads to a 404.

**Review by anderson-joyle — CHANGES_REQUESTED** [review]
Thanks for the additions!!! Please check my comments in the review.

File: README.md (hunk 1 above)
Comment (TrudAX) [reply README.md]: Thanks. What link is wrong? I checked all 3, they are all good

Comment (anderson-joyle) [reply README.md]: https://www.examtopics.com/exams/microsoft/mb-300/
Screenshot from 2021-11-22 07-01-49

If I go back to the home page, Microsoft is not listed as a provider.
If I try to search by MB-*, the engine suggests some Microsoft certification but they lead to a no-content page.

Am I missing something?

Comment (TrudAX) [reply README.md]: Interesting. It seems they are blocking the whole Microsoft based content from certain locations. 
added a note about this

**Review by anderson-joyle — APPROVED** [review]
Merging the changes.

**Pull request merged**